brave/src/main/java/brave/internal/JsonEscaper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.internal;
+
+// Initially, a copy of zipkin2.internal.JsonEscaper
+public final class JsonEscaper {
+  public static CharSequence jsonEscape(CharSequence v) {
```

Comment:
Should be a pretty simple win to accept `StringBuilder` as argument here instead of allocating inside.

Reply:
good point!